.genx/prompts/ui.js

```diff
@@ -24,8 +25,27 @@ module.exports = async (inquirer, prevAns = {}) => {
       when: !prevAns.fdc3,
       default: prevAns.fdc3 || false,
     },
+    {
+      name: 'fdc3ListenChannel',
+      type: 'confirm',
+      message: 'Listen to an fdc channel config in json format',
+      when: !prevAns.fdc3ListenChannel,
+      default: prevAns.fdc3ListenChannel || false,
+    },
```

Comment:
we'll need to rejig these prompts a bit, but I can do this once it's merged into my branch